### DISCUSSION TITLE
Will you provide a JRE?

### DISCUSSION THREAD

**Discussion by kwheat2064** [microsoft/openjdk#6] [Q&A] [answered]
Will there be JRE releases similar to what AdoptOpenJDK provides on their releases page? Some of our applications do not require the full JDK.

**Comment by brunoborges** [comment, marked as answer]
Hey @kwheat2064, thanks for the question.

Java 11 runtimes can be created as part of an application's build/CICD pipeline using `jlink`. 

We would love to know more about possible limitations or challenges in using that tool.

**Reply by Dmole** [reply]
If one is going to take the trouble to change old code to be jlink compatible one might as well go all the way and use Graal-native.

**Reply by karianna** [reply]
@Dmole, curious about your use case here.  When you say old code, are you referring to a pre-Java 11 codebase?  Java 11 code should run just fine on the appropriately jlinked runtime.

**Comment by kwheat2064** [comment]
Thanks! I've updated our pipeline to build a custom runtime with jlink.
